src/core/triple/literals.pl

```diff
@@ -337,6 +338,21 @@
     ->  atom_string(L,String)
     ;   L = String).
 
+:- dynamic normalise_warning_/0.
+
+set_normalise_warning :-
+    assertz(normalise_warning_).
+
+reset_normalise_warning :-
+    retractall(normalise_warning_).
+
+normalise_triple(rdf(X,P,Y,G),rdf(XF,P,YF)) :-
+    (   normalise_warning_
+    ->  true
+    ;   format(user_error,'Warning: ignoring graph ~q~n', [G]),
```

Comment:
Please use `json_log_warning_formatted` instead so it works properly with different logging backends we set up.